guides/common/modules/proc_enabling-client-connections-to-satellite.adoc

```diff
@@ -9,31 +9,31 @@ For more information on the ports used, see {InstallingServerDocURL}satellite-po
 include::snip_firewalld.adoc[]
 
 .Procedure
-. To open the ports for client to {Project} communication, enter the following command on the base operating system that you want to install {Project} on:
+ifdef::katello,satellite,orcharhino[]
+. Open the ports for clients on {ProjectServer}:
++
+[options="nowrap"]
+----
+# firewall-cmd --permanent \
+--add-port="5647/tcp" \
+--add-port="8000/tcp" \
+--add-port="9090/tcp"
+----
+endif::[]
+. Allow access to services on {ProjectServer}:
 +
 [options="nowrap"]
 ----
-# firewall-cmd \
+# firewall-cmd --permanent \
 --add-service=dns \
 --add-service=dhcp \
 --add-service=tftp \
 --add-service=http \
 --add-service=https \
-ifdef::katello,satellite,orcharhino[]
---add-port="5647/tcp" \
---add-port="8000/tcp" \
---add-port="9090/tcp" \
-endif::[]
 ifndef::katello,satellite,orcharhino[]
 --add-service=foreman-proxy \
 endif::[]
 --add-service=puppetmaster
 ----
-. Make the changes persistent:
-+
-[options="nowrap"]
-----
-# firewall-cmd --runtime-to-permanent
-----
```

Comment:
This part is incorrect. You either make changes to the permanent config, or the runtime config. Your current approach only applies it to the permanent config and doesn't apply it to runtime.

Reply:
I have added `firewall-cmd --permanent` instead, for ports and services both. Do you want me to restore these removed lines?

Reply:
@ekohl ^^ What solution do you suggest instead?

Reply:
My understanding of firewalld is that it's not the same. See https://firewalld.org/documentation/configuration/runtime-versus-permanent.html for more information, but please restore it as it previously was.

Reply:
@ekohl How about setting it as permanent and then reloading the service? That would work too, no?

Reply:
@Lennonka I don't know firewalld well enough to answer. To me it sounds like the right thing but it sounds like something you'd do across the entire documentation and not in this PR.

Perhaps there's some internal guidance within Red Hat / RHEL.

Reply:
My understanding is that `firewall-cmd --runtime-to-permanent` does not require the `firewall-cmd --reload` but `firewall-cmd --permanent` does (Lennonka's suggestion).

After careful consideration, we should just split ports and services into individual sets here IMO and restore them in the previous state (Ewoud's suggestion). 

CC: @sayan3296

Reply:
+1 
It makes sense to use `--permanent` while adding ports\services in firewalld, followed by `firewall-cmd --reload`. That's rather a widely acceptable and familiar approach for many system-admins.